docfx_yaml/extension.py

```diff
@@ -1897,6 +1964,21 @@ def convert_module_to_package_if_needed(obj):
         markdown_utils.remove_unused_pages(
             added_pages, app.env.moved_markdown_pages, normalized_outdir)
 
+    # Add summary pages as the second entry into the table of contents.
+    pkg_toc_yaml.insert(
+        1,
+        {
+            "name": f"{app.config.project} APIs",
+            "items": [
+                {"name": "Classes", "href": "summary_class.yml"},
+            ],
+        }
+    )
+    cgc_url = (
```

Comment:
Nit: Move this below writing the toc file.

Reply:
Done.